h2gis-functions/src/main/java/org/h2gis/functions/spatial/properties/ST_Distance_Sphere.java

```diff
@@ -0,0 +1,37 @@
+/**
+ * H2GIS is a library that brings spatial support to the H2 Database Engine
+ * <http://www.h2database.com>. H2GIS is developed by CNRS
+ * <http://www.cnrs.fr/>.
+ *
+ * This code is part of the H2GIS project. H2GIS is free software;
+ * you can redistribute it and/or modify it under the terms of the GNU
+ * Lesser General Public License as published by the Free Software Foundation;
+ * version 3.0 of the License.
+ *
+ * H2GIS is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License
+ * for more details <http://www.gnu.org/licenses/>.
+ *
+ *
+ * For more information, please consult: <http://www.h2gis.org/>
+ * or contact directly: info_at_h2gis.org
+ */
+
+package org.h2gis.functions.spatial.properties;
+
+/**
+ * For geometry type returns minimum distance in meters between two geometries
+ * @author Michael MATUR
+ */
+public class ST_Distance_Sphere extends ST_DistanceSphere {
```

Comment:
Why this alias ? Just to be compliant with an old postgis version ?

Reply:
This alias would be handy to test functions like this, because in MySQL it is `st_distance_sphere`

```kotlin
    @Query("FROM table r WHERE st_distance_sphere(r.point, :point) <= :distance")
    fun findAllByDistance(point: Point, distance: Double): List<Entity>
```